server/routes/confirmedOnReport/confirmedOnReportChangeReport.ts

```diff
@@ -27,16 +27,10 @@ export default class ConfirmedOnReportChangeReportRoutes {
     )
 
     return res.render(`pages/confirmedOnReport`, {
```

Comment:
Doesn't all this deleting mean that the above code isn't needed any more?

Maybe even `reportedAdjudicationsService .getConfirmationDetailsChangedReport` isn't required?